scripts/test_solidity_lsp.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3.8
+
+import argparse
+import json
+import os
+import subprocess
+
+from typing import List, Union, Any
+from deepdiff import DeepDiff
+
+# {{{ JsonRpcProcess
+class MyEncoder(json.JSONEncoder):
+    """
+    Encodes an object in JSON
+    """
+    def default(self, o): # pylint: disable=E0202
```

Comment:
```suggestion
    def default(self, o): # pylint: disable=method-hidden
```

But do we actually need this? The false-positive seems to have been fixed in pylint 2 years ago: https://github.com/PyCQA/pylint/issues/414.